spec/student_spec.rb

```diff
@@ -29,7 +32,8 @@
   end
 
   describe "#create_table" do
-    it 'creates the students table in the database' do
+    it 'creates the students table in the database', :skip_before do
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Hmm, that's a good question. It seems to work with and without the "IF EXISTS" portion of the query. My guess is because without that part of the query the test will always drop the table for that test and find the students created table in the following line. I will add in the "IF EXISTS" to be explicit.